examples/simulation_pytorch/README.md

```diff
@@ -0,0 +1,68 @@
+# Single-Machine Simulation of Federated Learning Systems
+
+This example is part of a series of blog posts. It's recommended to read the [blog post](https://flower.dev/blog/2021-01-14-single-machine-simulation-of-federated-learning-systems) before reading further.
+
+## Quick Start
+
+If you have docker on your machine you can execute this simulation using it. Start with building the container
+
+```shell
+docker build -t flower_federated_learning_simulation_pytorch .
+```
+
+and afterwards simply start the simulation in docker using
+
+```shell
+docker run --ipc=host -it --rm flower_federated_learning_simulation_pytorch
+```
+
+## Project Setup
+
+Start by cloning the example project. We prepared a single-line command that you can copy into your shell which will checkout the example for you:
+
+```shell
+git clone --depth=1 https://github.com/adap/flower.git && mv flower/examples/simulation_pytorch . && rm -rf flower && cd simulation_pytorch
+```
+
+This will create a new directory called `simulation_pytorch` containing the following files:
+
+```shell
+-- dataset.py
+-- Dockerfile
+-- pyproject.toml
+-- README.md
+-- run.sh
+-- simulation.py
```

Comment:
```suggestion
-- SimpleNet.py
-- simulation.py
```

Reply:
@tanertopal Updated them. Thanks and Cheers :)